packages/opencensus-core/src/metrics/gauges/types.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Metric, TimeSeries, Timestamp} from '../export/types';
+
+export interface Meter {
```

Comment:
Is this used anywhere in the PR? Is it for a future feature?

Reply:
Good catch, implemented by Gauge class. Also, this interface is required for DerivedGauge class and metric collector/producer (in future).